### DISCUSSION TITLE
Fantastic library

### DISCUSSION THREAD

**Discussion by jsjf93** [mantinedev/mantine#3036] [Kind words]
I've nothing constructive to say, but I've been using this library for a startup and the developer experience has been fantastic. I've migrated the project away from other packages for handling modals and forms and everything is working smoothly.

Awesome effort, thank you for all the work 🎉

**Comment by rtivital** [comment]
Thanks for kind words!

**Comment by colindotfun** [comment]
agreed -- very nice to use and very well written 🤓

**Comment by datner** [comment]
The iteration rate is first rate, most large component libraries are so slow moving, but you guys are amazingly fast!
This is invaluable if a user wants to become a power-user and heavily rely on mantine.

Thanks for enabling rapid progress!
